2996_reflection/reflection.md

```diff
@@ -4701,7 +4705,7 @@ consteval u8string_view u8identifier_of(info r);
 * [#.#]{.pnum} Otherwise, if `r` represents a class type, then either the typedef name for linkage purposes or the identifier introduced by the declaration of the represented type.
 * [#.#]{.pnum} Otherwise, if `r` represents an entity, `$typedef-name$`, or namespace alias, then the identifier introduced by the the declaration of what is represented by `r`.
 * [#.#]{.pnum} Otherwise, if `r` represents a base class specifier, then the identifier introduced by the declaration of the type of the base class.
-* [#.#]{.pnum} Otherwise (if `r` represents a description of a declaration of a non-static data member), then the identifier that would be introduced by the declaration of a data member having the properties represented by `r`.
+* [#.#]{.pnum} Otherwise (if `r` represents a description of a declaration of a non-static data member), then letting `$o$` be a `data_member_options_t` value such that `data_member_spec(type_of(r), $o$) == $r$` and `$o$.name->$is-utf8$()` is `true`, the contents of `$o$.name->$get-u8string$()` encoded with `$E$`.
```

Comment:
This needs to conditionally call `get-string` or `get-u8string` (based on `is-utf8`).

Also can you mix and match? For instance, If I constructed the name as `u8"hello"` can I get `identifier_of` that or does it have to be `u8identifier_of`?

Reply:
Yes, I think you should be able to mix and match.

Reply:
And nope, no need to be conditional - because you can mix and match! Equality of `data_member_spec` reflections is based on whether they produce the same data members - so there are potentially multiple such `data_member_options_t`-values that produce equivalent `data_member_spec`-reflections. That lets me kinda just say, "Give me the one for which `name` holds a `u8string`."